tests/Process/Builder/ProcessBuilderTest.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+namespace Infection\Tests\Process\Listener;
+
+use Infection\Mutant\Mutant;
+use Infection\Process\Builder\ProcessBuilder;
+use Infection\TestFramework\AbstractTestFrameworkAdapter;
+use PHPUnit\Framework\TestCase;
+
+
+class ProcessBuilderTest extends TestCase
```

Comment:
tests with Mockery should contain 

```php
protected function tearDown()
{
    Mockery::close();
}
```

as other tests do.

Unfortunately, we still can't use Mockery's [test listener](https://github.com/mockery/mockery/blob/master/library/Mockery/Adapter/Phpunit/TestListener.php) because it does not support PHPUnit 6+, so we have to do it manually